mithril-aggregator/src/database/repository/signed_entity_store.rs

```diff
@@ -310,4 +333,91 @@ mod tests {
         assert_eq!(records_to_update, updated_records);
         assert_eq!(expected_records, stored_records);
     }
+
+    #[tokio::test]
```

Comment:
Shall we test request conditions in this file (check if record are correctly filtered) ?

Reply:
These tests are now performed at a lower level with the new implementation.